enhancements/sig-policy/66-delete-policy-resources/README.md

```diff
@@ -0,0 +1,224 @@
+    conditions:
+    - lastTransitionTime: "2022-06-09T19:27:13Z"
+      message: pods [demo] in namespace default found as specified, therefore this Object template is compliant
+      reason: K8s `must have` object already exists
+      status: "True"
+      type: notification
+  compliant: Compliant
+  lastEvaluated: "2022-06-09T19:27:53Z"
+  lastEvaluatedGeneration: 1
+  relatedObjects:
+  - compliant: Compliant
+    object:
+      apiVersion: v1
+      kind: pods
+      metadata:
+        name: demo
+        namespace: default
+    properties:
+      createdByParent: true //delete when parent configPolicy is deleted
+    reason: Resource found as expected
+```
+### Scalability
+The extra fields that will be needed to store data for this change should not impact scalability much, as we
+would only add a few boolean fields to each configurationPolicy. Deleting a policy will take a bit longer if
+it is placed on a large amount of managed clusters, since the controller will now have to remove all created
+objects from each managed cluster in addition to the configurationPolicy. However, this would still be much
+faster than it would take a user to clean up those resources otherwise - without this change, the easiest
+way for users to clean up resources created by a policy would be to create another policy to delete the
+objects after deleting the original policy, so this could end up helping with scalability because users
+will not have to create as many policies on the hub cluster.
+
+### Open Questions
+
+### Test Plan
+
+- Unit tests can be implemented on some of the helper functions that we add to the controller, but
+will not be able to test whether resources are being removed properly.
+- E2E/integration tests to ensure that resources are being removed when policies are deleted and
+resources persist when the spec field is not set
+
+### Graduation Criteria
+
+#### Alpha
+
+1. The proposal is reviewed and accepted
+2. Implementation is completed to support the primary goals
+3. Test cases developed to demonstrate that the user stories are fulfilled
+
+#### Beta
+
+1. E2E/Integration tests complete
+
+### Upgrade / Downgrade Strategy
+
+The `spec.removeChildrenOnDelete` field being optional means that the controller will remain
+backwards compatible with any older policies.
+
+## Implementation History
+
+N/A
+
+## Drawbacks
+
+- Users may accidentally delete resources if they are not familiar with how the new field works,
+although they will not be able to do so unless they set the field without checking the documentation
+on it.
+- Since this will only remove objects that are specified in the policy yaml, objects created elsewhere
+will not be deleted when the policy is deleted. This means the deletion will probably not work well
+for removing operators, since they can create extra resources on the cluster that need to be cleaned
+up.
+- Currently, the policy creation UI defaults to adding `clusterConditions` to the placementRule when
```

Comment:
The UI isn't open-source. Could you please rephrase this to "some policy tooling" or similar?